src/github/deployment.test.ts

```diff
@@ -4,8 +4,16 @@ import { getLogger } from "config/logger";
 import { envVars } from "config/env";
 import { sqsQueues } from "../sqs/queues";
 import { GITHUB_CLOUD_API_BASEURL, GITHUB_CLOUD_BASEURL } from "~/src/github/client/github-client-constants";
+import { saveDeploymentInfo } from "models/deployment-service";
+import deployment_status from "fixtures/deployment_status-basic.json";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
+import { when } from "jest-when";
+import { WebhookPayloadDeploymentStatus } from "@octokit/webhooks";
 
 jest.mock("../sqs/queues");
+jest.mock("models/deployment-service");
```

Comment:
Do we really need to mock it? Could we just check dynamodb?

Reply:
tests added.